tools/build_utils.py

```diff
@@ -276,16 +320,22 @@ def build_tensorflow(tf_version,
         os.environ[
             "CC_OPT_FLAGS"] = " -mcx16 -mssse3 -msse4.1 -msse4.2 -mpopcnt -mno-avx"
 
-    command_executor("./configure")
+    if (platform.system() == 'Windows'):
+        os.environ["TF_OVERRIDE_EIGEN_STRONG_INLINE"] = "1"
+        os.environ["CC_OPT_FLAGS"] = "/arch:AVX"
+        command_executor(["python", "configure.py"])
+    else:
+        command_executor("./configure")
+
+    cmd = ["bazel", "build"]
+
+    if (platform.system() != 'Windows'):
+        cmd.extend([
+            "--config=noaws",
```

Comment:
--config=opt seems to be missing after the change

Reply:
done, this flag is needed for windows too